documents/Libraries/pbrlib/gen-glsl/ogsfx/mx_lighting_functions.glsl

```diff
@@ -41,7 +41,7 @@ vec3 getLightColor(int index)
 }
 
 float getLightIntensity(int index)
```

Comment:
For consistency, maybe all functions in this file should get the mx_ prefix, since it was added to functions above.

Reply:
I think I just missed this. I'll fix this.